roamance/src/components/auth/SocialAuth.tsx

```
@@ -0,0 +1,39 @@
+  delay?: number;
+}
+
+export function SocialAuth({
+  text = 'or continue with',
+  delay = 0.8,
+}: SocialAuthProps) {
+  return (
+    <>
+      <div className="relative z-10">
+        <div className="absolute inset-0 flex items-center">
+          <div className="w-full border-t border-border/50"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)